File: project_examples/java/project_main.java
public class Main {
    public static void main(String[] args) {
        int[] items = {12, 18, 21, 9, 31};
        int total = 0;
        for (int value : items) total += value;
        System.out.println("Total: " + total);
        System.out.println("Average: " + (total / (double) items.length));
    }
}

File: project_examples/cpp/project_main.cpp
#include <iostream>
#include <vector>
int main() {
    std::vector<int> items = {12, 18, 21, 9, 31};
    int total = 0;
    for (int value : items) total += value;
    std::cout << "Total: " << total << '\n';
    std::cout << "Average: " << (total / static_cast<double>(items.size())) << '\n';
}
